sdk/samples/iot/README.md

```diff
@@ -193,7 +178,7 @@ To run the samples, ensure you have the following programs and tools installed o
 - If running a DPS sample: `paho_iot_provisioning_sample`, `paho_iot_provisioning_sas_sample`
   - Have an [Azure IoT Hub Device Provisioning Service (DPS)](https://docs.microsoft.com/azure/iot-dps/quick-setup-auto-provision) created.
 
-- If running a Plug and Play sample: `paho_iot_hub_pnp_sample`, `paho_iot_hub_pnp_component_sample`
+- If running a Plug and Play sample: `paho_iot_pnp_sample`.
```

Comment:
Seeing now that there WILL be a component sample and that you are leaving it for next commit.  Then I would leave this component sample in.  sorry for the confusion.
```suggestion
- If running a Plug and Play sample: `paho_iot_pnp_sample`, `paho_iot_pnp_component_sample`, `paho_iot_pnp_with_provisioning_sample`
```